packages/near-membrane-base/src/membrane.ts

```diff
@@ -123,14 +123,16 @@ export interface InitLocalOptions {
     instrumentation?: InstrumentationHooks;
 }
 
-// istanbul ignore next
-export function init(
+export type Init = (
     color: string,
     trapMutations: boolean,
-    supportFlags: SupportFlagsField = SupportFlagsField.None,
+    supportFlags: number,
     foreignCallableHooksCallback: HooksCallback,
     options?: InitLocalOptions
-): HooksCallback {
+) => HooksCallback;
+
+// istanbul ignore next
+export function createMembraneMarshall(): Init {
```

Comment:
The return type of createMembraneMarshall is `Init` but the return type of `init()` is not that. Kinda confusing, is there a better name for this than `Init`?

Reply:
well, it was Init, but changed to createHookCallback now... which makes more sense.